cmd/cronosd/config/config.go

```diff
@@ -2,6 +2,11 @@ package config
 
 import sdk "github.com/cosmos/cosmos-sdk/types"
 
+const (
+	NetworkMainet = iota + 1
```

Comment:
suggest change:  NetworkMainnet

Reply:
update: maybe just call `Mainnet`?

Reply:
removed, use a different upgrade plan name for testnet3.